src/types/types.ts

```diff
@@ -1167,6 +1167,74 @@ export interface NewApiKeyResponse extends ApiKeyObject {
   value: string;
 }
 
+// Viewer 3D
+
+export interface ListReveal3DNodeAncestors extends Cursor, Limit {}
+
+export interface RevealNode3D extends Node3D {
+  /**
+   * The sector the node is contained in.
+   */
+  sectorId: CogniteInternalId;
+}
+
+export interface RevealRevision3D extends Revision3D {
+  sceneThreedFiles: Versioned3DFile[];
+}
+
+/**
+ * The file ID of the data file for this resource, with multiple versions supported.
+ * Use /3d/files/{id} to retrieve the file.
+ */
+export interface Versioned3DFile {
+  /**
+   * Version of the file format.
+   */
+  version?: number;
```

Comment:
`version` and `fileId` is required. I designed the API for it

Reply:
allrighty